lib/v2/mihoyo/maintainer.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
     '/bbs/official/:gids/:type?/:page_size?/:last_id?': ['CaoMeiYouRen'],
     '/ys/:location?/:category?': ['nczitzk'],
+    '/sr/:location?/:num?': ['shinanory']
```

Comment:
```suggestion
    '/sr/:location?/:num?': ['shinanory']
    '/ys/:location?/:category?': ['nczitzk'],
```
Please insert new entries in alphabetical order.